Platform.Win32/Tests.Win32/Tests.Win32.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
```

Comment:
It is a lot clearer that people are selecting x86 or x64 builds because **they have to** - any time we can reduce the impression that we actually have a pure _AnyCPU_ product I think we should do so.